assets/src/application/services/toaster/ManualToaster.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Button, ButtonGroup } from '@chakra-ui/core';
+import { v4 as uuidv4 } from 'uuid';
+
+// import { useToaster, UseToasterProps } from '@infraServices/toaster';
+import useSystemNotifications from './useSystemNotifications';
+
+// const options: UseToasterProps = {
+// 	duration: 5000,
+// 	position: 'bottom-right',
+// };
+
+const ManualToaster = (): JSX.Element => {
+	const { error, info, loading, success, warning } = useSystemNotifications();
+
+	const loadingNotice = (event: React.MouseEvent): void => {
+		event.preventDefault();
+		loading({ message: 'loading...', key: uuidv4(), loading: true });
+	};
+
+	const errorNotice = (event: React.MouseEvent): void => {
+		event.preventDefault();
+		error({ message: 'error!!!' });
+	};
+
+	const infoNotice = (event: React.MouseEvent): void => {
+		event.preventDefault();
+		const message = 'info: ' + uuidv4() + '\n' + uuidv4() + uuidv4();
```

Comment:
I feel lost here :smiley:

Reply:
Same here, I don't know why it's like that? 😄

Reply:
this was just for testing, this will be removed prior to merge